packages/gatsby/src/redux/reducers/schema.ts

```diff
@@ -0,0 +1,13 @@
+import { ActionsUnion, IGatsbyState } from "../types"
+
+export const schemaReducer = (
+  state: IGatsbyState["schema"] = {} as IGatsbyState["schema"],
```

Comment:
oof, this feels dangerous. I get that it was pre-existing behavior.. but I wonder if we can just seed this an empty `GraphQLSchema`

Reply:
We could also reverse the typing for something like:

```ts
state: IGatsbyState["schema"] | {} = {}
```

Reply:
Asking this because it would be quickest way than an empty `GraphQLSchema` object that needs to be initialized with a query (according to the TS error I get doing `state: IGatsbyState["schema"] = new GraphQLSchema()`.

I will push a solution with the `| {}` but I'm happy to get more info about using the `GraphQLSchema` if you have a way to initialize it without queries :)